src/topics/topicActions.js

```diff
@@ -19,7 +19,7 @@ export const fetchTopics = (streamId: number) => async (dispatch: Dispatch, getS
   dispatch(initTopics(topics, streamId));
 };
 
-export const fetchTopicsForActiveStream = (narrow: Narrow) => async (
+export const fetchTopicsForStream = (narrow: Narrow) => async (
```

Comment:
Ah, thanks for this cleanup! You may find 5542b01da informative for what that was about.

Reply:
In commit message: summary line is too long. Main solution is to drop the old or the new name (or both, and say something like "Fix misleading "...ActiveStream" name.")